.github/workflows/request-approvers-pr-review.yaml

```diff
@@ -0,0 +1,21 @@
+name: Request PR review from signalfx/gdi-java-approvers
+
+on:
+  pull_request:
+    # TOTO: test
```

Comment:
s/TOTO/TODO/ ?

Reply:
🤦‍♂️ 

Well, it doesn't work anyway because of missing permissions...

Reply:
Fire removed